pkg/kotsadm/types/constants_test.go

```diff
@@ -0,0 +1,123 @@
+package types
+
+import (
+	"testing"
+
+	"gopkg.in/go-playground/assert.v1"
+)
+
+func Test_getKotsadmLabels(t *testing.T) {
+	tests := []struct {
+		name         string
+		labels       []map[string]string
+		expectLabels map[string]string
+	}{
+		{
+			name: "pass case with additional labels",
+			labels: []map[string]string{
+				{
+					"foo": "foo",
+				},
+			},
+			expectLabels: map[string]string{
+				"kots.io/kotsadm": "true",
+				"kots.io/backup":  "velero",
+				"foo":             "foo",
+			},
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			labels := GetKotsadmLabels(test.labels...)
+			assert.Equal(t, test.expectLabels, labels)
+		})
+	}
+}
+
+func Test_getTroubleshootLabels(t *testing.T) {
+	tests := []struct {
+		name         string
+		labels       []map[string]string
+		expectLabels map[string]string
+	}{
+		{
+			name: "pass case with additional labels",
+			labels: []map[string]string{
+				{
+					"foo": "foo",
+				},
+			},
+			expectLabels: map[string]string{
+				"troubleshoot.io/kind": "support-bundle",
+				"foo":                  "foo",
+			},
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			labels := GetTroubleshootLabels(test.labels...)
+			assert.Equal(t, test.expectLabels, labels)
+		})
+	}
+}
+
+func Test_mergeLabels(t *testing.T) {
+	tests := []struct {
+		name         string
+		labels       []map[string]string
+		expectLabels map[string]string
+	}{
+		{
+			name: "pass case with merge labels",
+			labels: []map[string]string{
+				{
+					"foo": "foo",
+				},
+				{
+					"bar": "bar",
+				},
+				{
+					"baz": "baz",
+				},
+			},
+			expectLabels: map[string]string{
+				"foo": "foo",
+				"bar": "bar",
+				"baz": "baz",
+			},
+		},
+		{
+			name: "pass case with merge troubleshoot and kotadm labels",
+			labels: []map[string]string{
+				GetKotsadmLabels(),
+				GetTroubleshootLabels(),
+			},
+			expectLabels: map[string]string{
+				"kots.io/kotsadm":      "true",
+				"kots.io/backup":       "velero",
+				"troubleshoot.io/kind": "support-bundle",
+			},
+		},
+		{
+			name: "pass case with merge troubleshoot and kotadm additional labels",
+			labels: []map[string]string{
+				GetKotsadmLabels(),
+				GetTroubleshootLabels(),
+			},
```

Comment:
This looks the same as the above case, should we add some additional labels as args to `GetKotsadmLabels(...)`?

Reply:
yes, we should  add some additional labels for testing merge labels. The test has been changed